x-pack/plugins/observability/public/pages/alerts/components/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './alerts_flyout';
```

Comment:
@ersin-erdal I was thinking if `alerts` is `alerts_flyout` is redundant here. Since flyout is already in an alerts folder, I am not sure if we want to keep alerts_flyout name or simply change it to flyout. For example severity_badge or workflow_status_filter don't use alerts in the beginning of the file name.

What do you think? Same comment goes for `alerts_search_bar` and `alerts_disclaimer`

Reply:
Yes, i agree with you.
Besides this, i think flyout should be on page level not in the grid. I'll create some more PR's for refactoring such as moving cellActions out from the grid component. I'll tackle this with the PR.